lib/compat.php

```diff
@@ -216,6 +216,52 @@ function gutenberg_add_rest_nonce_to_heartbeat_response_headers( $response ) {
 
 add_filter( 'wp_refresh_nonces', 'gutenberg_add_rest_nonce_to_heartbeat_response_headers' );
 
+/**
+ * Ensure that the wp-json index contains the `permalink_structure` setting as
+ * part of its site info elements.
+ *
+ * @see https://core.trac.wordpress.org/ticket/42465
+ *
+ * @param WP_REST_Response $response WP REST API response of the wp-json index.
+ * @return WP_REST_Response Response that contains the permalink structure.
+ */
+function gutenberg_ensure_wp_json_has_permalink_structure( $response ) {
```

Comment:
Is the (re-?)introduction of permalink structure intended? Guessing not by #6319.

Reply:
Nope. I think this was a bad merge conflict. Removed in 8830dd7fbc705d05ff154bf4895c00b76582a3ee